cmd/list.go

```diff
@@ -266,8 +285,33 @@ func (cooked cookedListCmdArgs) HandleListContainerCommand() (err error) {
 		}
 
 		if cooked.RunningTally {
-			fileCount++
-			sizeCount += object.size
+			if containsProperty(cooked.properties, versionId) {
```

Comment:
do we need to make a call and iterate over all the properties so many times? We do it 3 times in this one method